src/docs/08-operation-guide/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Operation Guide
```

Comment:
Change this to `Introduction`
This is the text that will show in the left side navigation.